pkg/oauthserver/authenticator/password/basicauthpassword/basicauthpassword.go

```diff
@@ -45,14 +45,16 @@ type RemoteUserData struct {
 	PreferredUsername string `json:"preferred_username"`
 	// Email is the end-User's preferred e-mail address. Optional.
 	Email string `json:"email"`
+	// Groups is the end-User's groups. Optional.
+	Groups []string `json:"groups"`
```

Comment:
is it safe to pull in groups from the JSON automatically?

Reply:
Why not ?

Reply:
This IDP would basically fetch groups without configuration (unlike the other ones which need some form of config change).  With the prefixing by default, this is likely ok.